app/data/index.js

```diff
@@ -5,18 +5,25 @@ const { databaseConfig } = require('../config')
 const modelPath = path.join(__dirname, 'models')
 const db = {}
 
-const sequelize = new Sequelize(databaseConfig.database, databaseConfig.username, databaseConfig.password, databaseConfig)
+const sequelize = new Sequelize(
+  databaseConfig.database,
+  databaseConfig.username,
+  databaseConfig.password,
+  databaseConfig
+)
 
 fs.readdirSync(modelPath)
-  .filter(file => {
-    return (file.indexOf('.') !== 0) && (file !== 'index.js') && (file.slice(-3) === '.js')
+  .filter((file) => {
```

Comment:
This is okay but a note to say that we don't have to use the brackets on a parameter for an arrow function if theres only one param. eg `file => {...}` is valid.

From the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#description):
_The parentheses can only be omitted if the function has a single simple parameter. If it has multiple parameters, no parameters, or default, destructured, or rest parameters, the parentheses around the parameter list are required._

Reply:
Will take note of this, but I removed them now anyway to help with remembering that. Thank you.